charts/privatebin/Chart.yaml

```
@@ -5,7 +5,7 @@ description: A Helm chart for installing PrivateBin
 name: privatebin
 home: https://privatebin.info/
 icon: https://raw.githubusercontent.com/PrivateBin/assets/master/images/preview/icon.png
-version: 0.12.1
+version: 0.13.0
```

This will need to go to 0.14.0 because of other prior merges.

```suggestion
version: 0.14.0
```